packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -139,60 +213,82 @@ export class VaFileInput {
   }
 
   render() {
-    const { label, name, required, accept, error, hint, uswds } = this;
+    const { label, name, required, accept, error, hint, uswds, file, uploadStatus, headerSize } = this;
 
     const text = this.getButtonText();
 
     if (uswds) {
-      const labelClass = classnames({
-        'usa-label': true,
-        'usa-label--error': error,
-      });
-      const inputClass = classnames({
-        'usa-file-input': true,
-        'usa-input--error': error,
-      });
       const ariaDescribedbyIds =
         `${hint ? 'input-hint-message' : ''} ${
           error ? 'input-error-message' : ''
         }`.trim() || null; // Null so we don't add the attribute if we have an empty string
+      const fileInputTargetClasses = `file-input-target ${this.error ? 'file-input-target-error' : ''}`.trim();
 
       return (
         <Host>
-          {label && (
-            <label htmlFor="fileInputButton" class={labelClass} part="label">
-              {label}
-              {required && (
-                <span class="usa-label--required">
-                  {' '}
-                  {i18next.t('required')}
-                </span>
-              )}
-            </label>
-          )}
+          {label && this.renderLabelOrHeader(label, required, headerSize)}
           {hint && (
-            <span class="usa-hint" id="input-hint-message">
-              {hint}
-            </span>
+            <div class="input-hint-message">{hint}</div>
           )}
-          <slot></slot>
-          <span id="input-error-message" role="alert">
+          <div id="input-error-message" role="alert">
             {error && (
               <Fragment>
-                <span class="usa-sr-only">{i18next.t('error')}</span>
-                <span class="usa-error-message">{error}</span>
+                <span class="sr-only">{i18next.t('error')}</span>
+                <span>{error}</span>
```

Comment:
`usa-error-message` also is not specific to file input. Do the VA designs diverge from the USWDS error styles? If not, can we continue to use the module that contains that class?

Reply:
I am using the USWDS styles for hint and error again